user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Tutorials/Tutorial_Apps_Managing_Pages.md

```diff
@@ -1,98 +1,130 @@
 ---
 uid: Tutorial_Apps_Managing_Pages
 ---
+
 # Managing the pages of an app
 
-This tutorial shows how to add, change duplicate and remove pages in a low-code app.
+This tutorial shows how to add, change, duplicate, and remove pages in a low-code app.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.3.11.
+
+Expected duration: 5 minutes.
 
 > [!NOTE]
 > A low-code app will only show the navigation sidebar if there are at least two pages. If an app has only one page, the sidebar is not displayed outside of edit mode.
 
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.6 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
```

Comment:
I understand the difference although I believe it will be confusing to mention a different DataMiner version here.

Reply:
If everything looks the same in 10.3.6, I can just add 10.3.6 as the version at the top as well. I mentioned this version in the prerequisites because this is when duplicating a page is introduced.